### PR TITLE
docs(frontend):Guía markdown 179

### LINKED ISSUE #179: Actualización Guía Markdown
Agregar el contenido  de la sección de Markdown Guide, a la versión en Español.

Revisar:
- https://github.com/orgs/eoscostarica/projects/3#card-52322121

### PR DESCRIPTION
**Pull request by steph222**
### GH Issue
Resolves #179  

### Steps to test
1. Go to Tutoriales/Markdown Guía
2.  Check content is right.
3.  Compare content with Markdown Guide

#### CheckList
- [ ] It is similar to the Markdown Guide in English
- [ ] The content is adequate

### CODE REVIEW

File: docs/tutoriales/guia-markdown.md [hunk 1]
```diff
@@ -13,24 +13,23 @@ Para mayor información puede dar click en el siguiente link [¿Qué es Markdown
...
+| [Encabezado](https://www.markdownguide.org/basic-syntax/#headings) |  # H1   <br /> ## H2 <br />  ### H3  |
+| [Tipografía Negrita](https://www.markdownguide.org/basic-syntax/#bold) |  `**Texto**` or `__texto__` |
+| [Tipografía Itálica](https://www.markdownguide.org/basic-syntax/#italic) |   `*texto*` or  `_texto_`|
+| [Citas en Bloque](https://www.markdownguide.org/basic-syntax/#blockquotes-1)    | > Citas en bloque  |
+| [Lista Numérica](#ordered-list) | 1. Primer Item <br /> 2. Segundo Item <br /> 3. Tercer Item  |
+| [Lista no Ordenada](#unordered-list) | - Primer Item <br /> - Segundo Item <br /> - Tercer Item |
+| [Linea Horizontal](#horizontal-rule) | --- |
+| [Enlaces](https://www.markdownguide.org/basic-syntax/#links) | `[título](https://www.example.com)` |  
+| [Imagen](https://www.markdownguide.org/basic-syntax/#images-1) | `![texto](image.jpg)` |
+|[Salto de línea](#line-break) | texto + doble espacio |
```
Comment (leisterfrancisco) [inline docs/tutoriales/guia-markdown.md:27]: I think, to have the same uppercase first letter format, in the phrase _Salto de línea_, the word could be **L**ínea

Comment (steph222) [reply docs/tutoriales/guia-markdown.md]: Thanks!

Comment (leisterfrancisco) [reply docs/tutoriales/guia-markdown.md]: You welcome 🥇

**Review by xavier506 — APPROVED** [review]
LGTM

Note: lets update this branch with `upstream/master`

- `git checkout master`
- `git pull upstream/master`
- ` git checkout Guia....`
- `git merge master`

**Review by leisterfrancisco — APPROVED** [review]

**Pull request merged**